documentation/content/en/books/handbook/jails/_index.adoc

```diff
@@ -250,11 +250,17 @@ There are two ways to configure jails.
 The first one is to add an entry for each jail to the file [.filename]#/etc/jail.conf#.
 The other option is to create a file for each jail in the directory [.filename]#/etc/jail.conf.d/#.
 
-There is no right or wrong option.
-Each administrator must choose the one that best suits their needs.
-
 In case a host system has few jails, an entry for each jail can be added in the file [.filename]#/etc/jail.conf#.
-If the host system has many jails, it is good idea to have one configuration file for each jail in the [.filename]#/etc/jail.conf.d/# directory.
+If the host system has many jails, it is a good idea to have one configuration file for each jail in the [.filename]#/etc/jail.conf.d/# directory.
+
+The files you put in the [.filename]#/etc/jail.conf.d/# directory must have `.conf` as their extension.
+
+If you enabled jails in [.filename]#/etc/rc.conf# you should put the following into [.filename]#/etc/jail.conf#:
```

Comment:
"While it's not forbidden to use 'you', it might be better to use it less in this context."
See https://docs.freebsd.org/en/books/fdp-primer/book/#writing-style-be-clear

Reply:
Thanks a lot for the hint. I wasn't aware of that document. Cool. I will fix the text.